libraries/zookeeper_service.rb

```diff
@@ -86,6 +86,7 @@ class ZookeeperService < Chef::Provider
       include PoiseService::ServiceMixin
 
       def action_enable
+        new_resource.notifies(:restart, new_resource, :delayed)
```

Comment:
Either way we are comfortable with kicking zookeeper even if a directory owner changes?